datahub-web-react/src/graphql/glossaryTerm.graphql

```diff
@@ -42,5 +44,13 @@ query getGlossaryTerm($urn: String!, $start: Int, $count: Int) {
                 value
             }
         }
+         schemaMetadata(version: 0) {
+            ...schemaMetadataFields
+        }
+        previousSchemaMetadata: schemaMetadata(version: -1) {
+            ...schemaMetadataFields
+        }
```

Comment:
we don't need this since we aren't displaying schema version history

Reply:
removed